### PR TITLE
Added new meshery/meshery-perf project to Community Handbook

### LINKED ISSUE #2763: Add meshery/meshery-perf to Community Handbook Repository Overview
**Current Behavior**
The repo at github.com/meshery/meshery-perf is new and is not yet listed in the [Community Handbook's repository overview.](https://layer5.io/community/handbook/repository-overview)

**Desired Behavior**
Create an entry for the new `meshery/meshery-perf` repo in the repository overview.

---

<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by subhradwip01**
**Description**
Added new meshery/meshery-perf project to Community Handbook Repository Overview
This PR fixes #2763

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3479a780e0f58a6793635a39022c2ac76fb120c6 at: https://627f4be952fefc33a19e58fb--layer5ng.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -246,6 +246,12 @@ export const repo_data = [
         description:"GitHub Action to run SMP Performance Benchmarks on CI/CD pipelines.",
         repository:"https://github.com/layer5io/meshery-smp-action"
       },
+      {
+        project:"meshery-pref",
```
Comment (Chadha93) [inline src/sections/Community/Handbook/repo-data.js:250]:
```suggestion
        project:"Meshery-Perf",
```

**Comment by l5io** [comment]
🚀 Preview for commit a5c8c83374473738c5bf4f7b579ee1339353bf11 at: https://627f785e413ffe4e07c549aa--layer5ng.netlify.app

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**